asabiris/orchestration/sendmail.py

```diff
@@ -53,7 +52,7 @@ async def send_mail(
 		# Render a body
 		body_html, email_subject_body = await self.render(body_template, body_params)
```

Comment:
Mithun, you changed the way how you look into the library only when handling attachments, but you must do the same for body template as well.